packages/components/src/simple-select-control/style.scss

```
@@ -0,0 +1,122 @@
+.woocommerce-simple-select-control__dropdown {
+	position: relative;
+	width: 100%;
+	height: 56px;
+	border: 1px solid #b0b5b8;
+	box-shadow: none;
+	box-sizing: border-box;
+	border-radius: 3px;
+	background: #fff;
+	font-size: 16px;
+	line-height: 54px;
+	font-weight: 400;
+	margin-top: $gap;
+	margin-bottom: $gap;
+
+	.woocommerce-simple-select-control__selector {
+		&.components-button {
+			border: 0;
+			background: transparent;
+			box-shadow: none;
+			justify-content: unset;
+			margin-left: 0;
+			position: absolute;
+			top: 0;
+			left: 8px;
+			width: calc(100% - 24px);
+
+			&:focus:not(:disabled) {
+				box-shadow: none;
+				outline: none;
+				background-color: transparent;
+				outline-offset: initial;
+			}
+
+			&::after {
+				display: block;
+				pointer-events: none;
+				position: absolute;
+				float: right;
+				line-height: 56px;
+				font-family: dashicons, sans-serif;
+				font-size: 20px;
+				content: '\f140';
+				z-index: 101;
+				height: 24px;
+				width: 24px;
+				margin-top: 0;
+				top: 0;
+				right: 0;
+				bottom: 16px;
+				color: #000;
+			}
+		}
+	}
+
+	.woocommerce-simple-select-control__dropdown-content {
+		&.components-popover.is-bottom {
+			margin-top: -$gap-largest * 2;
+			z-index: 999;
+
+			&::before,
+			&::after {
+				display: none;
+			}
+		}
+
+		.components-button {
+			display: block;
+			position: relative;
+			padding: 10px 20px 10px 40px;
+			width: 100%;
+			text-align: left;
+
+			&.is-selected {
+				background: $muriel-gray-100;
+			}
+		}
+	}
+
+	&.is-empty {
+		.woocommerce-simple-select-control__selector {
+			&.components-button {
```

One optional nitpick if we want to the left and right sides to also be clickable:

```suggestion
			&.components-button {
				width: 100%;
				left: 0;
				padding-left: $gap;
				padding-right: $gap;
```

But we may have to adjust the pseudo after arrow with `right: $gap;` if we do this.